driver/pace/driver/run.py

```diff
@@ -310,6 +266,41 @@ def _initialize_dycore_state(
         return dycore_state
 
 
+@dataclasses.dataclass
+class TranslateConfig(InitializationConfig):
+    """
+    Configuration for pre-existing dycore state
+    """
+
+    dycore_state: Any
+    grid: Grid
```

Comment:
If we keep this config, could you decouple it from the Grid class? It's alright to use the Grid since we're only using this for tests (and that's where we've said we'll use Grid), but it's a fairly small change to make it more general purpose.

One way (the smallest change I can see) is to pass in `GridData`, `DampingCoefficients`, and `DriverGridData` instead. Another would be to pass in a DriverState instead of the DycoreState and Grid, and put the init_zeros calls for the other class in the context that calls this.